boards/arm/efr32_thunderboard/board.cmake

```diff
@@ -4,6 +4,10 @@ if(CONFIG_BOARD_EFR32BG22_BRD4184A)
 board_runner_args(jlink "--device=EFR32BG22C224F512IM40" "--reset-after-load")
 include(${ZEPHYR_BASE}/boards/common/jlink.board.cmake)
 
+elseif(CONFIG_BOARD_EFR32BG22_BRD4184B)
+board_runner_args(jlink "--device=EFR32BG22C224F512IM40" "--reset-after-load")
+include(${ZEPHYR_BASE}/boards/common/jlink.board.cmake)
```

Comment:
These are a copy of what's defined for BRD4184A. Let's combine them in the if clause, i.e.:
```if(CONFIG_BOARD_EFR32BG22_BRD4184A OR CONFIG_BOARD_EFR32BG22_BRD4184B)```

Reply:
@fkokosinski ok, done!